tutorials/builtin_algorithms/GCP_AI_Platform_Image_Classification_Built_in_Algorithm.ipynb

```diff
@@ -0,0 +1,604 @@
+{
```

Comment:
<p>Should that be <code>efficientnetb4</code> as opposed to <code>efficientetb6</code> in the <code>MODEL_TYPE</code> line?</p><p> </p><p>Please consider breaking this down into at least two cells.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ml-on-gcp/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>